service/counter/message.go

```diff
@@ -13,10 +12,12 @@ import (
 	"github.com/traPtitech/traQ/model"
 )
 
-var messagesCounter = promauto.NewCounter(prometheus.CounterOpts{
-	Namespace: "traq",
-	Name:      "messages_count_total",
-})
+// Do not initialize messagesCounter until DB operation is completed
+// /api/metrics will not provide traq_messages_count_total until all previous messages are counted
+var (
+	messagesCounter prometheus.Counter
+	once            sync.Once
+)
```

Comment:
以下の様にチャネルも定義して、初期化の完了を他のゴルーチンに通知する事で上手く実装出来そうです。

```suggestion
var (
	messagesCounter prometheus.Counter
	initOnce        sync.Once
	initDone        = make(chan struct{})
)
```